proto/enums/enums.proto

```diff
@@ -307,6 +307,8 @@ enum StatusCode {
     STATUSCHECK_CONTAINER_WAITING_UNKNOWN = 503;
     // Container event reason unknown
     STATUSCHECK_UNKNOWN_EVENT = 509;
+    // Status Check internal error
+    STATUSCHECK_INTERNAL_ERROR = 514;
```

Comment:
Could we make this a general Skaffold internal error and not just statuscheck?  Then we could capture `panic()` and notify about these too.